downstream/assemblies/platform/assembly-gs-platform-admin.adoc

```diff
@@ -13,9 +13,17 @@ To get started as an administrator, see the following:
 * xref:con-gs-manage-RBAC[Managing user access with role-based access control]
 
 include::platform/proc-gs-logging-in.adoc[leveloffset=+1]
+
+include::platform/proc-adding-a-subscription.adoc[leveloffset=+1]
+
```

Comment:
This feels like it's a duplicate of the Activating Red Hat Ansible Automation Platform details, just presented with slightly different wording/structure. Ideally, you just call in that file: assembly-aap-activate.adoc.